Client/Ecosia/UI/EmptyBookmarksView.swift

```diff
@@ -0,0 +1,269 @@
+        }
+        containerStackView.addArrangedSubview(spacerOne)
+        
+        addSection(imageNamed: "bookmarkAdd", text: .localized(.bookmarksEmptyViewItem0))
+        addSection(imageNamed: "exportShare", text: .localized(.bookmarksEmptyViewItem1), listItems: [
+            .localized(.bookmarksEmptyViewItem1NumberedItem0),
+            .localized(.bookmarksEmptyViewItem1NumberedItem1)
+        ])
+        
+        let buttonsContainerCenterView = UIView.build { centerView in
+            let buttonsContainer = UIView.build { view in
+                view.addSubview(self.learnMoreButton)
+                view.addSubview(self.importBookmarksButton)
+                NSLayoutConstraint.activate([
+                    self.learnMoreButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+                    self.importBookmarksButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+                    self.learnMoreButton.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+                    self.learnMoreButton.trailingAnchor.constraint(equalTo: self.importBookmarksButton.leadingAnchor, constant: -UX.InBetweenSpacerWidth).priority(.defaultHigh),
+                    self.importBookmarksButton.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+                    view.heightAnchor.constraint(equalToConstant: UX.ButtonAreaHeight)
+                ])
+            }
+            centerView.addSubview(buttonsContainer)
+            NSLayoutConstraint.activate([
+                buttonsContainer.centerXAnchor.constraint(equalTo: centerView.centerXAnchor),
+                buttonsContainer.bottomAnchor.constraint(equalTo: centerView.bottomAnchor),
+                buttonsContainer.topAnchor.constraint(equalTo: centerView.topAnchor)
+            ])
+        }
+        
+        containerStackView.addArrangedSubview(buttonsContainerCenterView)
+        
+        // setup buttons
+        learnMoreButton.addTarget(self, action: #selector(onLearnMoreTapped), for: .touchUpInside)
+        importBookmarksButton.addTarget(self, action: #selector(onImportTapped), for: .touchUpInside)
+
+        applyTheme()
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(applyTheme), name: .DisplayThemeChanged, object: nil)
+    }
+    
+    private func addSection(imageNamed: String, text: String, listItems: [String]? = nil) {
+        // first section (tap the bookmark icon when you find a page you want to share)
+        let sectionStackView = UIStackView()
+        sectionStackView.axis = .horizontal
+        sectionStackView.alignment = .top
+        
+        if traitCollection.userInterfaceIdiom == .pad {
+            sectionStackView.addArrangedSubview(createSpacerView(width: UX.SectionSpacerWidth))
+        }
+        
+        let sectionIcon = UIImageView()
+        sectionIcon.tintColor = .theme.ecosia.secondaryText
```

Comment:
I would guess this line is even obsolete as `applyTheme()` will also set it, right?